src/MICore/CommandFactories/lldb.cs

```diff
@@ -192,5 +197,33 @@ public async override Task<string> VarAssign(string variableName, string express
                 return await base.VarAssign(variableName, value, threadId, frameLevel);
             }
         }
+
+        private bool? _requiresOnKeywordForBreakInsert;
+        private const string OldLLDBMIVersionString = "lldb-350.99.0";
+
+        // In LLDB 3.5, -break-insert -f requires a string before the actual method name.
+        // We use a placeholder 'on' for this.
+        // Later versions do not require the 'on' keyword.
+        public override async Task<bool> RequiresOnKeywordForBreakInsert()
+        {
+            if (!_requiresOnKeywordForBreakInsert.HasValue)
+            {
+                _requiresOnKeywordForBreakInsert = false;
```

Comment:
>_requiresOnKeywordForBreakInsert = false; [](start = 16, length = 41)

If we wind up trying to set two breakpoints at the same time this is going to do the wrong thing. I think you should only assign once you know the answer.